app/assets/javascripts/admin/orders/controllers/orders_controller.js.coffee

```diff
@@ -1,26 +1,41 @@
-angular.module("admin.orders").controller "ordersCtrl", ($scope, $compile, $attrs, shops, orderCycles) ->
```

Comment:
This file looks a bit confusing in the diff, but basically `ordersCtrl` was renamed to `orderCtrl` (above) and the contents of this file are new.